pkg/commands/workload_test.go

```diff
@@ -73,6 +76,10 @@ func TestWorkloadCommand(t *testing.T) {
 }
 
 func TestWorkloadOptionsValidate(t *testing.T) {
+	localRepo := filepath.Join(string(filepath.Separator), "path", "to", "local", "repo")
```

Comment:
can `string(filepath.Separator)` be a global const/var to not cal `string(...)` each time?